fw.lua

```diff
@@ -174,7 +174,7 @@ local function _ac_lookup(self, needle, haystack, ctx)
 	local match, _ac
 
 	-- dictionary creation is expensive, so we use the id of
-	-- the rule as the key to cache the created dictionary 
+	-- the rule as the key to cache the created dictionary
```

Comment:
I appreciate getting rid of an extra space, but I prefer to keep commits atomic and as small as possible